config/database.yml

```diff
@@ -24,6 +24,8 @@ default: &default
 development:
   <<: *default
   database: rails_recipe_app_development
+  username: postgres
+  password: pw4Admin
```

Comment:
It can be better if we keep the databases config file without any username and password because those are going to be different from one computer to another.